frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -58,6 +60,33 @@ interface MetadataDisplayProps {
 export function MetadataDisplay(props: MetadataDisplayProps) {
   const { resource, extraRows } = props;
 
+  function makeOwnerReferences(ownerReferences: KubeOwnerReference[]) {
```

Comment:
Could drop the `make` part here - since it's basically a component it should be Capitalized.